IPs/RSKIP428.md

```diff
@@ -0,0 +1,97 @@
+---
+rskip: 428
+title: Emit a new event with the utxo outpoints when a peg-out is created
+created: 23-APR-24
+author: NC
+purpose: Sca
+layer: Core 
+complexity: 2
+status: Draft
+description: 
+---
+
+|RSKIP          | 428                |
+| :------------ |:-------------------|
+|**Title**      | Emit a new event with the utxo outpoints when a peg-out is created |
+|**Created**    | 23-APR-24          |
+|**Author**     | NC                 |
+|**Purpose**    | Sca                |
+|**Layer**      | Core               |
+|**Complexity** | 2                  |
+|**Status**     | Draft              |
+
+## Abstract
+
+This RSKIP proposes creating a new event that stores the outpoint values of the inputs used in a 
+peg-out to enable the PowHSM sign segwit peg-outs.
+
+## Motivation
+
+For the PowHSM sign segwit peg-outs[1], the outpoint values must be provided since they are needed 
+for calculating the sighash of a segwit bitcoin transaction. Currently, selected UTXOs for a 
+peg-out are removed from the available utxos set once the peg-out is created. Storing the outpoint 
+values in an event, once a peg-out is created and before the selected utxos are discarded, is a 
+solution that will allow the PowPeg to retrieve and provide the required outpoint values to the 
+PowHSM to be able to sign a given segwit peg-out.
+
+## Specification
+
+The proposal is to create a new event to store the bitcoin transaction hash of the peg-out and 
+the outpoint values of each utxo used as input of the given peg-out. Then this event will be emitted
+whenever a transaction that spends from the federation address, in other words peg-outs, is created.
```

Comment:
` in other words **a peg-out**` (singular, not plural, since you are talking about **a** transaction)